ariadne/contrib/tracing/opentracing.py

```diff
@@ -62,6 +66,10 @@ async def resolve(
     def filter_resolver_args(
         self, args: Dict[str, Any], info: GraphQLResolveInfo
     ) -> Dict[str, Any]:
+        for key, val in args.items():
+            if isinstance(val, (cgi.FieldStorage, UploadFile)):
+                args[key] = TraceableFile(val)
```

Comment:
This should be recursive because GraphQL inputs can be nested.

It also looks like we are modifying `args` in place. We can't do that.

I'm thinking we should implement custom `safe_copy_args` function that would return deep copy of arguments and swap uploaded files with uploaded file's repr. It could default to `str()` for types other than lists/dicts/bool/numeric.